src/import/load-costume.js

```diff
@@ -211,8 +216,14 @@ const loadCostumeFromAsset = function (costume, runtime, optVersion) {
             typeof costume.rotationCenterY === 'number' && !isNaN(costume.rotationCenterY)) {
         rotationCenter = [costume.rotationCenterX, costume.rotationCenterY];
     }
-    if (costume.asset.assetType.runtimeFormat === AssetType.ImageVector.runtimeFormat) {
-        return loadVector_(costume, runtime, rotationCenter, optVersion);
+    if (costume.asset.assetType === AssetType.ImageVector) {
+        return loadVector_(costume, runtime, rotationCenter, optVersion)
+            .catch(() => {
+                // Use default asset if original fails to load
+                costume.assetId = runtime.storage.defaultAssetId.ImageVector;
```

Comment:
Is it possible for defaultAssetId to be null in storage? It looks like it's {} by default. If setting that is required, it may be better to log an error below, so it's easier to catch (say line 257 where we check that storage exists)

Reply:
@fsih good question. Taking a look at [scratch-storage](https://github.com/LLK/scratch-storage/blob/29daf3f503f11a30a41106e7f5db0bc10bf26758/src/ScratchStorage.js#L16), the `defaultAssetId` data is populated soon after it's initialized to {}, so I think it's unlikely it will be null intentionally. However, that makes sense to me to add an error log in case this process fails for some reason. I added the error log a little later in the load costume process since not having the default asset isn't crucial until then. Let me know though if you think it should happen earlier.

Reply:
I think it makes sense to log the error sooner, because then if an issue comes up we will see it, rather than needing to hit another error before we discover that one.